bin/whenever

```diff
@@ -20,6 +20,9 @@ OptionParser.new do |opts|
     options[:clear] = true
     options[:identifier] = identifier if identifier
   end
+  opts.on('-a', '--clear-all-flags-crontab') do |identifier|
```

Comment:
opts.on takes a third argument which is the user-visible documentation string for this option. Include one here.

We don't use the block argument, so I think you should remove it from this block.